conf/routes

```diff
@@ -0,0 +1,17 @@
+# Routes
+# This file defines all application routes (Higher priority routes first)
+# ~~~~
+
+GET         /                     io.pathfinder.controllers.ViewController.index
```

Comment:
index returns the webpage, and the top part is an autogenerated comment that describes the file, its fine the way it is